docs/components/exosphere.md

```diff
@@ -0,0 +1,11 @@
+# Exosphère
+
+Exosphère is a reimplementation of TrustZone (TZ), also known at Nintendo’s Secure Monitor (Secure_Monitor.bin). It has the highest privilege mode available on the Switch’s processor, and has access to everything on the console.
```

Comment:
=> of Nintendo's Secure Monitor.

To be pedantic, TrustZone is the name of Arm's technology segregating the secure world and the insecure world.

Same remarks below